<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="109" started-at="2020-01-02T06:48:07 IST" finished-at="2020-01-02T06:48:07 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="109" started-at="2020-01-02T06:48:07 IST" finished-at="2020-01-02T06:48:07 IST">
      <class name="TestNG.TestData1">
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:TestNG.TestData1@5427c60c]" name="m1" duration-ms="16" started-at="2020-01-02T06:48:07 IST" data-provider="LoginDataProvider" finished-at="2020-01-02T06:48:07 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:TestNG.TestData1@5427c60c]" name="m1" duration-ms="15" started-at="2020-01-02T06:48:07 IST" data-provider="LoginDataProvider" finished-at="2020-01-02T06:48:07 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[user2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="PASS" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:TestNG.TestData1@5427c60c]" name="m1" duration-ms="0" started-at="2020-01-02T06:48:07 IST" data-provider="LoginDataProvider" finished-at="2020-01-02T06:48:07 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[user3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
        <test-method status="FAIL" signature="m1(java.lang.String, java.lang.String)[pri:0, instance:TestNG.TestData1@5427c60c]" name="m1" duration-ms="0" started-at="2020-01-02T06:48:07 IST" data-provider="LoginDataProvider" finished-at="2020-01-02T06:48:07 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Rahul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rahul123]]>
              </value>
            </param>
          </params>
          <exception class="CustomExceptions.InvalidUserorPasswordException">
            <full-stacktrace>
              <![CDATA[CustomExceptions.InvalidUserorPasswordException
at TestNG.TestData1.m1(TestData1.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- CustomExceptions.InvalidUserorPasswordException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- m1 -->
      </class> <!-- TestNG.TestData1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
